dbms/src/Flash/EstablishCall.cpp

```diff
@@ -37,6 +37,11 @@ EstablishCallData::EstablishCallData(AsyncFlashService * service, grpc::ServerCo
 EstablishCallData::~EstablishCallData()
 {
     GET_METRIC(tiflash_object_count, type_count_of_establish_calldata).Decrement();
+    if (stopwatch)
```

Comment:
also need to update `EstablishConnectionSyncOrAsync` to not update these metrics for async connection.

Reply:
I think even if `stopwatch` is nullptr, you should update `tiflash_coprocessor_handling_request_count`

Reply:
okk